src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1422,6 +1437,12 @@ private bool CanShowNewPresetStateDialog(object parameter)
             return DynamoSelection.Instance.Selection.Count > 0;
         }
 
+        public IEnumerable<NodeModel> GetInputNodes()
```

Comment:
This method is badly named. It doesn't give any info about from which collection the nodes are drawn from. Are we talking about nodes in the current selection? I'd consider inlining it or making it `private` and making it more descriptive.